chutney/ui/src/app/modules/backups/components/backups-admin.component.html

```diff
@@ -1,59 +1,66 @@
 <div class="container-fluid">
-    <div class="row">
-        <form class="w-100" *ngIf="backupForm">
-            <table class="table table-striped table-hover">
-                <thead>
-                <tr>
-                    <th scope="col" class="text-center" width="100px"></th>
-                    <th scope="col" class="text-center">{{ 'admin.backups.table.time.title' | translate }}</th>
-                    <th scope="col" class="text-center"
-                    *ngFor="let backupable of backupables"
-                    >{{ 'admin.backups.table.'+backupable+'.title' | translate }}</th>
-                </tr>
-                </thead>
-                <tbody>
-                <tr class="bg-light">
-                    <td class="text-center align-middle">
-                        <button type="button" class="btn btn-sm btn-primary"
-                                title="{{ 'global.actions.add' | translate }}"
-                                (click)="launchBackup()"
-                                [disabled]="!isOneBackupSelected()">
-                            <span class="fa fa-plus" aria-hidden="true"></span>
-                        </button>
-                    </td>
-                    <td></td>
-                    <td scope="row" class="text-center align-middle"
```

Comment:
My Intellij tells me that "scope="row" is obsolete

Reply:
Yes in html5 we can set scope only for <th>.
We have to pass over html code to migrate from v4 to v5